conf/app.routes

```diff
@@ -436,3 +436,8 @@ POST       /:draftId/asset/:index/change-shares-on-stock-exchange
 
 GET       /:draftId/asset/:index/check-share-answers                           controllers.shares.ShareAnswerController.onPageLoad(index : Int, draftId : String)
 POST      /:draftId/asset/:index/check-share-answers                           controllers.shares.ShareAnswerController.onSubmit(index : Int, draftId : String)
+
+GET        /:draftId/asset/:index/what-is-the-property-land-address                        controllers.WhatIsThePropertyOrLandAddressController.onPageLoad(mode: Mode = NormalMode, index: Int, draftId: String)
```

Comment:
url's should followthe pattern:
 /:draftId/asset/:index/property-or-land/uk-address

Reply:
complete, can't include "or", keyword in scala